ckanext/depositar_theme/templates/snippets/search_form.html

```diff
@@ -0,0 +1,9 @@
+## This override is for fixing https://github.com/ckan/ckan/issues/8736
```

Comment:
```suggestion
{# This override is for fixing https://github.com/ckan/ckan/issues/8736 #}
```
Use the correct [syntax](https://jinja.palletsprojects.com/en/stable/templates/#comments) to write comments or the string will be shown on the page.